src/assets/thirdPartyServices/snet/text_generation/index.js

```diff
@@ -12,147 +12,94 @@ import InfoIcon from "@material-ui/icons/Info";
-import PhilipPlaitAvatar from "../../../images/ThirdPartyServices/snet/text_generation/PhilipPlait_avatar.jpg";
-
-import RebeccaSkloot from "../../../images/ThirdPartyServices/snet/text_generation/RebeccaSkloot.jpg";
-import RebeccaSklootAvatar from "../../../images/ThirdPartyServices/snet/text_generation/RebeccaSkloot_avatar.jpg";
-
-import RichardDawkins from "../../../images/ThirdPartyServices/snet/text_generation/RichardDawkins.jpg";
-import RichardDawkinsAvatar from "../../../images/ThirdPartyServices/snet/text_generation/RichardDawkins_avatar.jpg";
-
-import RickyGervais from "../../../images/ThirdPartyServices/snet/text_generation/RickyGervais.jpg";
-import RickyGervaisAvatar from "../../../images/ThirdPartyServices/snet/text_generation/RickyGervais_avatar.jpg";
+const imgPath = (name, orgId = "snet", serviceId = "text-generation", extension = "jpg") => {
+  const correctedServiceId = serviceId.replace(/\-/g, "_");
+  const trimmedName = name.replace(/[\s\.\'']/g, "");
+  return `${process.env.REACT_APP_SNET_CDN}/assets/images/ThirdPartyServices/${orgId}/${correctedServiceId}/${trimmedName}.${extension}`;
+};
 
-import SamHarris from "../../../images/ThirdPartyServices/snet/text_generation/SamHarris.jpg";
-import SamHarrisAvatar from "../../../images/ThirdPartyServices/snet/text_generation/SamHarris_avatar.jpg";
+const avatarPath = (name, orgId = "snet", serviceId = "text-generation", extension = "jpg") => {
+  const avatarName = name + "_avatar";
+  return imgPath(avatarName, orgId, serviceId, extension);
+};
 
-import TerenceMcKenna from "../../../images/ThirdPartyServices/snet/text_generation/TerenceMcKenna.jpg";
-import TerenceMcKennaAvatar from "../../../images/ThirdPartyServices/snet/text_generation/TerenceMcKenna_avatar.jpg";
+const defaultImgPath = imgPath("DefaultImage", undefined, undefined, "png");
+
+const runNamesWithoutMedia = [
+  { key: "universal", value: "Universal Generator" },
+  { key: "badastronomer", value: "Phil Plait" },
+  { key: "barackobama", value: "Barack Obama" },
+  { key: "beebrookshire", value: "Bethany Brookshire" },
+  { key: "berniesanders", value: "Bernie Sanders" },
+  { key: "billgates", value: "Bill Gates" },
+  { key: "conanobrien", value: "Conan O'Brien" },
+  { key: "deborahblum", value: "Deborah Blum" },
+  { key: "deepakchopra", value: "Deepak Chopra" },
+  { key: "elonmusk", value: "Elon Musk" },
+  {
+    key: "ericrweinstein",
+    value: "Eric Weinstein",
+    image: imgPath("EricWeinstein", undefined, undefined, "png"),
+    avatar: avatarPath("EricWeinstein", undefined, undefined, "png"),
+  },
+  { key: "hillaryclinton", value: "Hillary Clinton" },
+  { key: "jimmyfallon", value: "Jimmy Fallon" },
+  { key: "joebiden", value: "Joe Biden" },
+  {
+    key: "joerogan",
+    value: "Joe Rogan",
+    image: imgPath("JoeRogan", undefined, undefined, "png"),
+    avatar: avatarPath("JoeRogan", undefined, undefined, "png"),
```

Comment:
You are always passing `undefined` for  orgId and serviceId. Can we remove both these params from `imgPath` and `avatarPath` methods. They seem redundant.